source/main/modules/window-manager/index.ts

```diff
@@ -121,7 +128,16 @@ export default class WindowManager {
           callingWindow.minimize()
           break
         case 'win-close':
-          callingWindow.close()
+          if (process.platform !== 'darwin') {
+            const showInNotification = Boolean(global.config.get('system.showInNotification'))
```

Comment:
The config is called `leaveAppRunning`. The config is called the same both on Windows and MacOS, even though the MacOS preference page displays different labels: `Leave app running in the notification area` (Windows/Linux) and `Show app in the notification area` (MacOS)